tenable/ot/assets.py

```diff
@@ -58,24 +59,10 @@ def list(self,
         Example:
             >>>     for asset in tot.assets.list(limit=500):
                         print(asset)
-        '''
+        """
         if not sort:
-            sort = [{'field': 'id', 'direction': 'DescNullLast'}]
-
-        query_variables = {
-            'search': search,
-            'sort': sort,
-            'startAt': start_at,
-            'limit': limit
-        }
-        if filter:
-            query_variables['filter'] = filter
+            sort = [{"field": "id", "direction": "DescNullLast"}]
 
-        graph_obj = GraphObject(
-            object_name=ASSETS_QUERY_OBJECT_NAME,
-            query=ASSETS_QUERY,
-            resp_schema=AssetsSchema,
-            query_variables=query_variables
+        return super().list(
```

Comment:
Not sure if it is required, but maybe add support for assets.asset('asset_id') (very similay to  ot.plugins.plugin('Plugin_id') support) with built in query filter (I think it is very useful case...) WDYT?

Reply:
done